src/framework/preferences/js/Panels.js

```diff
@@ -143,7 +133,8 @@ var fluid_2_0 = fluid_2_0 || {};
      * this is all done, the onDomBind event is fired.
      */
     fluid.prefs.subPanel.resetDomBinder = function (that) {
-        that.container = $(that.container.selector);
+        var userJQuery = that.container.constructor;
```

Comment:
This is unusual - was this change required by a bug that was discovered? This code is copied from the framework and should be marked with a TODO to be eliminated once we have the new renderer